src/components/ContentWidgets/TodoList/TodoListEditor.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { WidgetEditorProps } from '../../ContentMapping/ContentMapping';
+
+/**
+ * TodoListEditor creates an editable widget to change what is displayed in the TodoList
+ * 
+ * Last Modified
+ * Nitesh Chetry
+ * July 3, 2019
+ */
+
+export const TodoListEditor: React.FC<WidgetEditorProps> = ({
+    editedContent,
+    setEditedContentOnChange
+}) => {
+    const list = editedContent.todoList_items || [];
+    let newList = [...list]
```

Comment:
I would actually have this within the onClick functions, because if you don't actually modify this list, it's using up extra memory. Small optimization, but it's an easy one to fix.